scripts/downstream/create-downstream-for-courage.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -xe
+
+setup_service node v14.18.0
+setup_service yarn 1.21.1
+
+# download okta-ui artifact version if empty and assign to upstream_artifact_version
+if [[ -z "${upstream_artifact_version}" ]]; then
+  local upstream_publish_receipt="";
+  pushd ${OKTA_HOME}/okta-ui > /dev/null
+    download_job_data global publish_receipt upstream_publish_receipt okta-ui ${upstream_artifact_sha}
+  popd > /dev/null
+  echo "okta-ui publish receipt: ${upstream_publish_receipt}"
```

Comment:
upstream_publish_receipt will this always be a courage package? 
Can you please add some examples as comments.